pkg/scheduler/apis/config/types.go

```diff
@@ -88,6 +88,16 @@ type KubeSchedulerConfiguration struct {
 	// If this value is nil, the default value will be used.
 	BindTimeoutSeconds *int64
 
+	// PodInitialBackoffSeconds is the initial backoff for unschedulable pods.
+	// If specified, it must be greater than 0. If this value is null, the default value (1s)
+	// will be used.
+	PodInitialBackoffSeconds *int64 `json:"podInitialBackoffSeconds"`
```

Comment:
@liggitt Internal version shouldn't have `"json:..."` tag, right?

(not sure it's mandatory or optional, but I notice most internal types.go don't have the tag)

Reply:
ah, true. good catch

Reply:
thanks, this was removed